lib/results_validators/persons_validator.rb

```diff
@@ -28,6 +28,15 @@ class PersonsValidator < GenericValidator
     LETTER_AFTER_PERIOD_WARNING = "'%{name}' is missing a space after a period in their name. Please confirm that this is indeed the correct spelling or fix the name."
     SINGLE_LETTER_FIRST_OR_LAST_NAME_WARNING = "'%{name}' has a single letter as first or last name. Please fix the name or confirm that this is indeed the competitor's correct name according to an official document."
     SINGLE_NAME_WARNING = "'%{name}' has only one name. Please confirm that this is indeed the competitor's full name according to an official document."
+    UNKNOWN_DOB_WARNING = "Some unknown DOB warning is thrown, please check with WST."
+
+    DOB_VALIDATION_ISSUES = [
+      :dob_jan_one,
+      :dob_very_young,
+      :dob_not_so_young,
+    ].freeze
+
+    DOB_VALIDATION_ISSUES_MAP = DOB_VALIDATION_ISSUES.to_h { |issue| [issue, issue] }
```

Comment:
Why do you force the values into a hash with themselves?

Reply:
To access them with ease - like `DOB_VALIDATION_ISSUES_MAP[:dob_jan_one]`. But anyway I've removed these lines, so most probably won't be relevant anymore.